service/src/main/java/de/starwit/mapper/AppTemplateMapper.java

```diff
@@ -1,39 +1,41 @@
-package de.starwit.mapper;
```

Comment:
rejected because of formatting

Reply:
line endings changed to CRLF
![image](https://user-images.githubusercontent.com/10485792/161916960-02e764fa-752e-4d3b-ab1e-a41658288bf6.png)